File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/DomainOrder.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Ordering metadata for domain registration and transfers.
  /// </summary>
  [DataContract]
  public class DomainOrder {
    /// <summary>
    /// Cost of Whois Privacy
    /// </summary>
    /// <value>Cost of Whois Privacy</value>
    [DataMember(Name="whoisPrivacyCost", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "whoisPrivacyCost")]
    public string WhoisPrivacyCost { get; set; }

    /// <summary>
    /// Gets or Sets Services
    /// </summary>
    [DataMember(Name="services", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "services")]
    public DomainOrderServices Services { get; set; }

    /// <summary>
    /// Map of TLDs to domain service IDs.
    /// </summary>
    /// <value>Map of TLDs to domain service IDs.</value>
    [DataMember(Name="tldServices", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "tldServices")]
    public Object TldServices { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DomainOrder {\n");
      sb.Append("  WhoisPrivacyCost: ").Append(WhoisPrivacyCost).Append("\n");
      sb.Append("  Services: ").Append(Services).Append("\n");
      sb.Append("  TldServices: ").Append(TldServices).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/domain_order.dart
part of swagger.api;

class DomainOrder {
  /* Cost of Whois Privacy */
  String whoisPrivacyCost = null;

  DomainOrderServices services = null;
/* Map of TLDs to domain service IDs. */
  Object tldServices = null;

  DomainOrder();

  @override
  String toString() {
    return 'DomainOrder[whoisPrivacyCost=$whoisPrivacyCost, services=$services, tldServices=$tldServices, ]';
  }

  DomainOrder.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    whoisPrivacyCost = json['whoisPrivacyCost'];
    services = new DomainOrderServices.fromJson(json['services']);
    tldServices = new Object.fromJson(json['tldServices']);
  }

  Map<String, dynamic> toJson() {
    return {
      'whoisPrivacyCost': whoisPrivacyCost,
      'services': services,
      'tldServices': tldServices
     };
  }

  static List<DomainOrder> listFromJson(List<dynamic> json) {
    return json == null ? new List<DomainOrder>() : json.map((value) => new DomainOrder.fromJson(value)).toList();
  }

  static Map<String, DomainOrder> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, DomainOrder>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new DomainOrder.fromJson(value));
    }
    return map;
  }
}
